auth-api/migrations/versions/d5b5eb2e8dd0_added_branch_name.py

```diff
@@ -0,0 +1,42 @@
+"""added branch name
+
+Revision ID: d5b5eb2e8dd0
+Revises: 69a7e464fef3
+Create Date: 2021-03-03 21:30:29.597634
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+from sqlalchemy.sql import column, table
+
+revision = 'd5b5eb2e8dd0'
+down_revision = '69a7e464fef3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('orgs', sa.Column('branch_name', sa.String(length=250), nullable=True))
+    op.add_column('orgs_version', sa.Column('branch_name', sa.String(length=250), nullable=True))
+
+    invitation_type_table = table('invitation_types',
+                                            column('code', sa.String(length=15)),
+                                            column('description', sa.String(length=100)),
+                                            column('default', sa.Boolean())
+                                            )
+
+    op.bulk_insert(
+        invitation_type_table,
+        [
+            {'code': 'GOVM', 'description': 'An invitation to activate a GOV Ministry account',
```

Comment:
May be confirm with BA if it's GOVM or GOVN, have heard both in different places.

Reply:
settled on GOVM itself as per BA.